test/types/connection.test.ts

```diff
@@ -20,6 +20,8 @@ conn.readyState === 99;
 
 expectError(conn.readyState = 0);
 
+expectError(conn.createCollections());
```

Comment:
Why is this expectError here?

Reply:
You said to write a test showing it could be used.

Reply:
`expectError()` means you expect the code within to cause a TypeScript compiler error. The problem isn't the test, just the `expectError()`.